apps/web/cypress/e2e/multi-dwelling/multi-dwelling-9.10.14-walkthrough.cy.ts

```diff
@@ -24,6 +29,19 @@ describe("multi dwelling: 9.10.14 walkthrough", () => {
     });
   });
 
+  it("should display all parts of breadcrumbs", () => {
+    cy.getByTestID(TESTID_BREADCRUMBS).should("be.visible");
+    cy.getByTestID(
+      GET_TESTID_BREADCRUMBS_BREADCRUMB(EnumBreadcrumbIds.HOME),
+    ).should("be.visible");
+    cy.getByTestID(
+      GET_TESTID_BREADCRUMBS_BREADCRUMB(EnumBreadcrumbIds.MULTI_DWELLING_UNIT),
+    ).should("be.visible");
+    cy.getByTestID(
+      GET_TESTID_BREADCRUMBS_BREADCRUMB(EnumBreadcrumbIds.LAST),
+    ).should("be.visible");
+  });
+
```

Comment:
checking each section of the breadcrumbs at a spot where we have a lot of sections